src/server/game/AI/SmartScripts/SmartScriptMgr.h

```diff
@@ -600,7 +600,8 @@ enum SMART_ACTION
     SMART_ACTION_SCENE_CANCEL                       = 130,    // sceneId
     // 131 - 135 : 3.3.5 reserved
     SMART_ACTION_PLAY_CINEMATIC                     = 135,    // reserved for future uses
-    SMART_ACTION_END                                = 136
+    SMART_ACTION_PLAY_SPELL_VISUAL_KIT              = 137,    // spellVisualKitId
```

Comment:
You skipped 136, are you reserving that number for something else?
Unless you simply overlooked the change and it should be
```cpp
    SMART_ACTION_PLAY_SPELL_VISUAL_KIT              = 136,    // spellVisualKitId
    SMART_ACTION_END                                = 137
```

Reply:
This is intended, #21336

Reply:
OK, thanks, I missed out on those details.
(https://github.com/TrinityCore/TrinityCore/pull/21336/files#diff-7644387f3507b6763048646e59512f28)